formatTest/typeCheckedTests/expected_output/imperative.re

```diff
@@ -55,12 +55,10 @@ try (
 };
 
 let result =
-  (
-    while false {
-      ()
-    }
-  ) ==
-    () ?
+  while false {
```

Comment:
Also, these parens were not needed. It makes the monadic bind code `>==` much more beautiful as well. For example, @samoht found that our function printing was far too verbose:

```rust
entry_of_file t index file mode sha1 blob >>= (
  fun | None => Lwt.return_unit 
      | Some e => e
)
```
(Imagine when they are nested, the extra parens really disrupt the flow).

But with the new logic, they will be printed much better (still not perfect, but progress!)

```rust
entry_of_file t index file mode sha1 blob >>=
fun | None => Lwt.return_unit 
    | Some e => e
```